frontend/src/composables/useRequestFee.ts

```diff
@@ -1,22 +1,28 @@
 import type { Ref } from 'vue';
-import { ref, watch } from 'vue';
+import { computed, ref, watch } from 'vue';
 
+import { useDebouncedTask } from '@/composables/useDebouncedTask';
 import { getRequestFee } from '@/services/transactions/request-manager';
 import { TokenAmount } from '@/types/token-amount';
 
 export function useRequestFee(
   rpcUrl: Ref<string | undefined>,
   requestManagerAddress: Ref<string | undefined>,
   requestAmount: Ref<TokenAmount | undefined>,
+  debounced?: boolean | undefined,
```

Comment:
Nitpick: for an optional parameter you don't need to add the undefined type.

Reply:
Ah, thanks for the info. I am pretty fresh in typescript, don't mind if you see this beginner mistakes :smiley:

Reply:
Yea all good. I just want to mention also such small things, so you can get a better feeling for typescript more quickly.